scripts/cdnCheck.js

```diff
@@ -0,0 +1,16 @@
+import fetch from "node-fetch";
```

Comment:
@prashantjagasia can you rename this file to use kebab-case? That way we can follow the existing conventions in this repo with how to do file names. Lets go with `sdk-logos-cdn-check.js`